crates/kubelet/src/pod/mod.rs

```diff
@@ -203,6 +223,33 @@ impl Pod {
         }
     }
 
+    /// Sets the status of all specified containers to Waiting, and optionally
+    /// sets the pod status message.
+    pub async fn initialise_status(
+        &self,
+        client: &kube::Client,
+        container_names: &Vec<&String>,
+        initial_message: Option<String>,
+    ) {
+        let mut all_waiting_map = HashMap::new();
+        for name in container_names {
+            let waiting = crate::container::Status::Waiting {
+                timestamp: chrono::Utc::now(),
+                message: "PodInitializing".to_owned(),
+            };
+            all_waiting_map.insert(name.to_string(), waiting);
```

Comment:
nit: I think `clone` is preferred to `to_string` in this case

Reply:
That was what I thought but it doesn't work because `clone` produces an `&String` and I need a `String`.